arithmetization/src/main/java/net/consensys/linea/zktracer/module/tables/instructionDecoder/InstructionDecoder.java

```diff
@@ -126,7 +125,7 @@ public void commit(List<MappedByteBuffer> buffers) {
       traceStackSettings(op, trace);
       traceBillingSettings(op, trace);
       trace
-          .opcode(Bytes.ofUnsignedInt(i))
+          .opcode(UnsignedByte.of(i))
```

Comment:
@DavePearce I've had to correct this a couple of times here and there. Just letting you know, not sure if it's a corset change that's at the source of this change in signature.